example_projects/simple-binary/run.sh

```diff
@@ -15,5 +15,6 @@ cargo with "echo {bin} {args}" -- run -- myargs
 cargo with "echo {bin} {args}" -- test -- myargs
 
 # Some examples which should fail
-! cargo with "echo {bin}"
-cargo with "echo {bin}" -- not-a-cargo-subcommand || echo "asdfsadfadf"
+(! cargo with "echo {bin}") && true || false
+(! cargo with "echo {bin}" -- not-a-cargo-subcommand) && true || false
+(! cargo with "some-command-that-does-not-exist" -- run) && true || false
```

Comment:
The parentheses and `&& true` aren’t needed; `! cargo with "echo {bin}" || false` would suffice.

Perhaps `! cargo with false -- run || false` should be added as a failing example, to test a command that’s successfully `exec()`ed but returns a nonzero status.